### DISCUSSION TITLE
diffence of using fork-url and rpc-url while testing

### DISCUSSION THREAD

**Discussion by selkoyuncu** [Cyfrin/foundry-full-course-cu#779] [Q&A]
while testing with "forge test" on Seplolia network , what happens if i test with "rpc-url" instead of "fork-url" 
output is same but  is there somethning goes on  different under the hood  

below is 2 different test command 

`forge test --match-test testPriceFeedIsAccurate --fork-url $SEPOLIA_RPC`
vs 
`forge test --match-test testPriceFeedIsAccurate --rpc-url $SEPOLIA_RPC`

**Comment by theirrationalone** [comment]
FORK: VIRTUAL Support, Whole network Disguised. Mocks all the behaviour of any live chain, You might be able to see fake tx to mainnet into your mempool (Alchemy) if provided.

RPC: REAL Support, Real Testnet. Forwards all txs to real testnet, Also you can see those txs into your mempool (Alchemy) if provided.


Our intention with FORK is, at least we can see how our protocol will work on any intended chain (Main | Test)net.
